TROUBLESHOOTING-GUIDE.md

```diff
@@ -34,3 +34,41 @@ These are **Apple error logs** logging *all* failed outgoing network connections
 When integrating the Thunderhead SDK manually into your app, you may encounter this compile error.
 
 In **Build Settings**, ensure the **Framework Search Paths** contains the framework filepath. If the framework is placed in your project directory, simply set the framework search path to $(SRCROOT) and set it to recursive.
+
+## How to resolve `UITableView` layout display issues
+
+If you come across issues with how the layout is rendered in your `UITableView`'s, try the following
+
+### Resolve `UITableView` layout issue by implementing `tableView:heightForRowAtIndexPath:`
+- Implement `tableView:heightForRowAtIndexPath:` delegate method in the view controller. The method implementation is required to enable the SDK swizzling implementation to call the original implementation.
```

Comment:
in all your view controllers where you are looking to display in-list Optimizations.

Reply:
Added